src/java/org/apache/cassandra/index/sai/README.md

```diff
@@ -0,0 +1,116 @@
+
+- The on-disk/SSTable index formats for both string and numeric data have been completely replaced. Strings are indexed
+  on disk using our proprietary on-disk byte-ordered trie data structure, while numeric types are indexed using Lucene's
+  balanced kd-tree.
+- While indexes continue to be managed at the column level from the user's perspective, the storage design at the column
+  index level is row-based, with related offset and token information stored only once at the SSTable level. This
+  drastically reduces our on-disk footprint when several columns are indexed on the same table.
+- The query path is synchronous and index searches run on IO threads.
+- Tracing, metrics, virtual table-based metadata and snapshot-based backup/restore are supported out of the box.
+
+Many similarities with standard secondary indexes remain:
+
+- The full set of C* consistency levels is supported for both reads and writes.
+- Index updates are synchronous with mutations and do not require any kind of read-before-write.
+- Queries are implemented on the back of C* range reads.
+- Paging is supported.
+- Only token ordering of results is supported.
+- Index builds are visible to operators as compactions and are executed on compaction threads.
+- All DML and DDL statements are CQL-based.
+- Single-node management operations are available via nodetool. (ex. stop & rebuild_index)
+
+## Quick Start
+
+The following short tutorial will get you up-and-running with storage-attached indexing.
+
+### Build and Start Cassandra
+
+Follow the instructions to build and start Cassandra in README.asc in root folder of the Cassandra repository
+
+### Create a Simple Data Model
+
+1.) Run the following DDL statements to create a table and two indexes:
+
+`CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy' , 'Cassandra': '1'};`
+
+`USE test;`
+
+`CREATE TABLE person (id int, name text, age int, PRIMARY KEY (id));`
+
+`CREATE CUSTOM INDEX ON person (name) USING 'StorageAttachedIndex' WITH OPTIONS = {'case_sensitive': false};`
+
+`CREATE CUSTOM INDEX ON person (age) USING 'StorageAttachedIndex';`
+
+2.) Add some data.
+
+`INSERT INTO person (id, name, age) VALUES (1, 'John', 21);`
+
+`INSERT INTO person (id, name, age) VALUES (2, 'john', 50);`
+
+`INSERT INTO person (id, name, age) VALUES (3, 'Boris', 43);`
+
+`INSERT INTO person (id, name, age) VALUES (4, 'Caleb', 34);`
+
+### Make Some Queries
+
+1.) Query for everyone named "John", ignoring case.
+
+`SELECT * FROM person WHERE name = 'John';`
+
+```
+ id | age | name
+----+-----+------
+  1 |  21 | John
+  2 |  50 | john
+```
+
+2.) Query for everyone between the ages of 18 and 25.
+
+`SELECT * FROM person WHERE age >= 18 AND age <= 35;`
+
+```
+ id | age | name
+----+-----+-------
+  1 |  21 |  John
+  4 |  34 | Caleb
+```
+
+## Contributors
+
+- Marc Selwan
+- Caleb Rackliffe
+- Zhao Yang
+- Jason Rutherglen
+- Maciej Zasada
+- Andrew de la Peña
```

Comment:
It's `Andres` instead of `Andrew`, although I like how `Andrew de la Peña` sounds :)

Reply:
My apologies, I copied the list from somewhere else and didn't read it properly